### ISSUE TITLE
Windows Black UnicodeDecodeError

### ISSUE THREAD

**Issue by flavienliger** [pre-commit/pre-commit-hooks#320]
Hi,

I can't create a simple pre-commit with Black. 
I'm on : 
- Windows 10
- python 3.6.5 (I tried 3.7.0 too)
- pre-commit 1.11.1

This is my .pre-commit-config.yaml
```yaml
repos:
-   repo: https://github.com/ambv/black
    rev: stable
    hooks:
    - id: black
      language_version: python3.6
```

And error log :
```
[WARNING] Unstaged files detected.
[INFO] Stashing unstaged files to C:\Users\DigitalDistrict/.cache\pre-commit\patch1539176976.
[INFO] Initializing environment for https://github.com/ambv/black.
[INFO] Installing environment for https://github.com/ambv/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Restored changes from C:\Users\DigitalDistrict/.cache\pre-commit\patch1539176976.
An unexpected error has occurred: CalledProcessError: Command: ('C:\\Users\\DigitalDistrict\\.cache\\pre-commit\\repohhwc_cd2\\py_env-python3.6\\Scripts\\pip.exe', 'install', '.')
Return code: 2
Expected return code: 0
Output:
    Processing c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2

Errors:
    Exception:
    Traceback (most recent call last):
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\site-packages\pip\_internal\basecommand.py", line 228, in main
        status = self.run(options, args)
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\site-packages\pip\_internal\commands\install.py", line 291, in run
        resolver.resolve(requirement_set)
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\site-packages\pip\_internal\resolve.py", line 103, in resolve
        self._resolve_one(requirement_set, req)
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\site-packages\pip\_internal\resolve.py", line 257, in _resolve_one
        abstract_dist = self._get_abstract_dist_for(req_to_install)
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\site-packages\pip\_internal\resolve.py", line 210, in _get_abstract_dist_for
        self.require_hashes
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\site-packages\pip\_internal\operations\prepare.py", line 324, in prepare_linked_requirement
        abstract_dist.prep_for_dist(finder, self.build_isolation)
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\site-packages\pip\_internal\operations\prepare.py", line 126, in prep_for_dist
        build_requirements, isolate = self.req.get_pep_518_info()
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\site-packages\pip\_internal\req\req_install.py", line 448, in get_pep_518_info
        pp_toml = pytoml.load(f)
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\site-packages\pip\_vendor\pytoml\parser.py", line 10, in load
        return loads(fin.read(), translate=translate, filename=getattr(fin, 'name', repr(fin)))
      File "c:\users\digitaldistrict\.cache\pre-commit\repohhwc_cd2\py_env-python3.6\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 867: character maps to <undefined>
    You are using pip version 10.0.1, however version 18.1 is available.
    You should consider upgrading via the 'python -m pip install --upgrade pip' command.
```

Thank you in advance for your time.
Regards.

**Comment by asottile** [comment]
this is the wrong project to report this to and a duplicate of https://github.com/pre-commit/pre-commit/issues/835

tl;dr: pip 10.x can't handle black, when a new `virtualenv` is released this will be magically fixed.  for now use black `18.6b4`.

**Comment by flavienliger** [comment]
I had a doubt, but it's because of pip...
Sorry I had looked, but not enough, and thank you for the answer.

Good day.
